ui-common/src/components/ServiceMapModal/ServiceMapModal.module.sass

```diff
@@ -1,64 +1,106 @@
 @import "../../variables.module"
```

Comment:
this code looks very similar to the CSS code you declared in #1055 
why do you need to declare it in both PRs?